package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "JSA-Emerald-Shenzhen",
+  "version": "1.0.0",
+  "description": "## Wiki",
+  "main": "index.js",
+  "dependencies": {
+    "prettier": "^1.19.1"
+  },
+  "devDependencies": {
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-prettier": "^6.5.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "eslint-plugin-react-hooks": "^1.7.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint '**/*.{js,jsx}' --ignore-pattern node_modules/",
+    "lint-fix": "eslint --fix '**/*.{js,jsx}' --ignore-pattern node_modules/",
+    "postinstall": "cp .hooks/* .git/hooks/"
```

@Jeffzholy you need to make sure the hook is executable here:
```suggestion
    "postinstall": "chmod +x ./hooks/* && cp .hooks/* .git/hooks/"
```